cmd/burrow/commands/config_options.go

```diff
@@ -10,102 +10,129 @@ import (
-			Desc:   "Validator index (in validators list - GenesisSpec or GenesisDoc) from which to set ValidatorAddress",
+		accountIndexOpt: cmd.Int(cli.IntOpt{
+			Name:   "i account-index",
+			Desc:   "Account index (in accounts list - GenesisSpec or GenesisDoc) from which to set Address",
 			Value:  -1,
-			EnvVar: "BURROW_VALIDATOR_INDEX",
+			EnvVar: "BURROW_ACCOUNT_INDEX",
 		}),
 
-		validatorAddressOpt: cmd.String(cli.StringOpt{
-			Name:   "a validator-address",
-			Desc:   "The address of the the signing key of this validator",
-			EnvVar: "BURROW_VALIDATOR_ADDRESS",
+		accountAddressOpt: cmd.String(cli.StringOpt{
+			Name:   "a account-address",
+			Desc:   "The address of the the signing key of this account",
+			EnvVar: "BURROW_ACCOUNT_ADDRESS",
 		}),
 
-		validatorPassphraseOpt: cmd.String(cli.StringOpt{
-			Name:   "p validator-passphrase",
-			Desc:   "The passphrase of the signing key of this validator (currently unimplemented but planned for future version of our KeyClient interface)",
-			EnvVar: "BURROW_VALIDATOR_PASSPHRASE",
+		accountPassphraseOpt: cmd.String(cli.StringOpt{
+			Name:   "p account-passphrase",
+			Desc:   "The passphrase of the signing key of this account (currently unimplemented but planned for future version of our KeyClient interface)",
+			EnvVar: "BURROW_ACCOUNT_PASSPHRASE",
 		}),
 
-		validatorMonikerOpt: cmd.String(cli.StringOpt{
-			Name:   "m validator-moniker",
-			Desc:   "An optional human-readable moniker to identify this validator amongst Tendermint peers in logs and status queries",
-			EnvVar: "BURROW_VALIDATOR_MONIKER",
+		accountMonikerOpt: cmd.String(cli.StringOpt{
+			Name:   "m account-moniker",
+			Desc:   "An optional human-readable moniker to identify this account amongst Tendermint peers in logs and status queries",
+			EnvVar: "BURROW_ACCOUNT_MONIKER",
 		}),
 
 		externalAddressOpt: cmd.String(cli.StringOpt{
 			Name:   "x external-address",
 			Desc:   "An external address or host name provided with the port that this node will broadcast over gossip in order for other nodes to connect",
 			EnvVar: "BURROW_EXTERNAL_ADDRESS",
 		}),
+
+		configFileOpt: cmd.String(cli.StringOpt{
+			Name:   "c config",
+			Desc:   "Use the specified burrow config file",
+			EnvVar: "BURROW_CONFIG",
+		}),
+
+		genesisFileOpt: cmd.String(cli.StringOpt{
+			Name:   "g genesis",
+			Desc:   "Use the specified genesis JSON file rather than a key in the main config, use - to read from STDIN",
+			EnvVar: "BURROW_GENESIS",
+		}),
+	}
+}
+
+func (opts *configOptions) initialize() (*config.BurrowConfig, error) {
+	conf, err := obtainBurrowConfig(*opts.configFileOpt, *opts.genesisFileOpt)
+	if err != nil {
+		return nil, fmt.Errorf("could not obtain config: %v", err)
+	}
+	err = opts.configure(conf)
+	if err != nil {
+		return nil, fmt.Errorf("could not configure config: %v", err)
 	}
+	return conf, nil
 }
 
 func (opts *configOptions) configure(conf *config.BurrowConfig) error {
```

Comment:
I think we can/should remove this entirely and make it a single shot in the function above